### ISSUE TITLE
Submitting app for Code Path Observer Group

### ISSUE THREAD

**Issue by ariel-ledesma** [CaptainCrunchTaco/SimpleTodo#2]
/cc @CoderRed99

**Comment by CoderRed99** [comment]
Very nice job. The YouTube video is fine for this submission, but did you get a chance to look at licecap?  :) Thanks!

**Comment by ariel-ledesma** [comment]
Thanks! I did look at licecap but that required use of the emulator (which I'm not a big fan of because of memory issues). I found a tutorial that shows you how to screen record your device using Terminal and command line. 

http://www.phonearena.com/news/How-to-record-your-screen-on-Android-4.4-KitKat_id49133

https://www.youtube.com/watch?v=9LFnhXXl3J4

It's sweet. I have to test and see if I can record .gif using this procedure, though. Still, I feel like this would be a good alternative for those who have a device but are working on computers with small amounts of memory, like me X(
